src/base/gps.h

```diff
@@ -55,6 +55,21 @@ class GPSTransform {
   std::vector<Eigen::Vector3d> XYZToEll(
       const std::vector<Eigen::Vector3d>& xyz) const;
 
+  std::vector<Eigen::Vector3d> EllToENU(
```

Comment:
It seems like that the convention is that the first point in the input list is considered as the origin/ref? It would seem more natural to require this point explicitly, equivalent to the ENUToXYZ method?

Reply:
Right, I have applied the modification.